app/models/service_provider_identity.rb

```diff
@@ -65,6 +65,10 @@ def happened_at
     last_authenticated_at.in_time_zone('UTC')
   end
 
+  def hide_change_email?
```

Comment:
cc @mdiarra3 I think we should align between this pull request and #11717 and have a single method, since this is being used for the same purpose.

Since this lives in the model, we should also avoid any UI-specific naming, so something like `requested_single_email?` similar to #11717 would be my vote.